virtool/samples/api.py

```diff
@@ -281,6 +292,7 @@ async def create(req):
         "user": {
             "id": user_id
         },
+        "labels": data.get("labels"),
```

Comment:
The `labels` field should always be an `list`. In this case, you are setting it to `None`. It should be an empty list.